app.py

```diff
@@ -1,13 +1,19 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests
+import os
+import trelloapp
+import json
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/') #redirect root to the url or route mapped to the index function
 def root():
-    return redirect(url_for('index'))
+    return redirect(url_for('getAll')) ## replaced index
 
+## All route from below up to module is NO LONGER applicable since replacing index 
+## with getAll from Module 2. Keep for reference No longer calling session.py
 @app.route('/items', methods = ["GET", "POST", "PUT"])
```

Comment:
This route (and other calling `session`) should be removed (as we're using trello to back the todos and not the flask session cookie)